docs/openj9_support.md

```diff
@@ -180,7 +180,7 @@ OpenJDK 17 binaries are expected to function on the minimum operating system lev
 | AIX 7.1 TL5                               | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |
 | AIX 7.2 TL4                               | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |
 
-:fontawesome-solid-bell:{: .warn aria-hidden="true"} **Important:** AIX OpenJ9 builds require the [XL C++ Runtime](https://www.ibm.com/support/pages/fix-list-xl-cc-runtime-aix).
+:fontawesome-solid-bell:{: .warn aria-hidden="true"} **Important:** AIX OpenJ9 builds require the [XL C++ Runtime](https://www.ibm.com/support/pages/fix-list-xl-cc-runtime-aix#161X).
```

Comment:
This doesn't help much. If I navigate to that page and select a different column in the table, following this link again doesn't switch to the 16.1.x column. On the other hand, it doesn't seem to hurt anything.

Reply:
At least opening the link originally navigates to the correct version. I've updated the wording as well to mention 16.1

Reply:
That wasn't my experience. I also tried the link in brave and firefox (each for the first time) and a new incognito window in chrome; each displayed the 17.1.x column.

Reply:
I'm not sure what more we can do about it, beyond bringing this to IBM's attention.